frame/contracts/src/wasm/runtime.rs

```diff
@@ -352,6 +354,7 @@ impl RuntimeCosts {
 			ReentrantCount => s.reentrance_count,
 			#[cfg(feature = "unstable-interface")]
 			AccountEntranceCount => s.account_reentrance_count,
+			InstantationNonce => s.instantiation_nonce,
```

Comment:
Above would imply it here as well

```suggestion
			#[cfg(feature = "unstable-interface")]
			InstantationNonce => s.instantiation_nonce,
```

Reply:
I want to merge https://github.com/paritytech/substrate/pull/12787  first so that is no longer necessary.  I suppose I opened this too early.

Reply:
Fair enough